vue-example/src/components/Arkose.vue

```diff
@@ -91,14 +99,14 @@ export default {
       console.log('Could not load the Arkose API Script!');
     };
   },
-  destroyed() {
+  unmounted () {
```

Comment:
hmm does this available in Vue2? I thought this was introduced in V3. Can you double check?

Reply:
Yeah `destroyed` is for v2. Looks like the linter that was added was for v2. Is there a reason this example isn't v3?

Reply:
I can change it to use v2 if we're happy to stay on Vue2